packages/components/card/src/card.tsx

```diff
@@ -44,23 +66,74 @@ const Card = (props: TCardProps) => (
       background: ${props.theme === 'dark'
         ? designTokens.colorNeutral95
         : designTokens.colorSurface};
-    `}
-    // Allow to override the styles by passing a `className` prop.
-    // Custom styles can also be passed using the `css` prop from emotion.
-    // https://emotion.sh/docs/css-prop#style-precedence
-    className={props.className}
-  >
-    {props.insetScale === 'none' ? (
-      // Use a `<div>` to ensure that there is always a wrapper container.
-      // This is mostly useful in case custom styles are targeting this element.
+      cursor: ${props.isDisabled
+        ? 'not-allowed'
+        : isClickable
+        ? 'pointer'
+        : 'default'};
+      pointer-events: ${props.isDisabled ? 'none' : 'auto'};
+      :hover {
+        background: ${props.theme === 'dark'
+          ? isClickable
+            ? designTokens.colorNeutral90
+            : undefined
+          : isClickable
+          ? designTokens.colorNeutral98
+          : undefined};
+      }
+      // Disables link text styling
+      color: inherit;
+      // Change the opacity of the content, not the card itself
+      & > div {
+        opacity: ${props.isDisabled ? 0.5 : 1};
+      }
+    `,
+    className: props.className,
+  };
+
+  const content =
+    props.insetScale === 'none' ? (
       <div>{props.children}</div>
     ) : (
       <Inset scale={props.insetScale} height="expanded">
         {props.children}
       </Inset>
-    )}
-  </div>
-);
+    );
+
+  if (props.to) {
+    if (props.isExternalLink) {
+      if (typeof props.to !== 'string') {
+        throw new Error(
+          '`to` must be a `string` when `isExternal` is provided.'
+        );
+      }
```

Comment:
We're used to be using the [`warning`](https://github.com/commercetools/ui-kit/blob/main/packages/utils/src/warning.ts#L17) helper for this cases (don't throw an error as it would very likely break the whole page where this component is used).
```suggestion
warning(
  typeof props.to === 'string',
  'ui-kit/Card: "to" property must be a string when "isExternal" value is true'
);
```

Reply:
Works for me! See fbf459afc2e4bb78bfbfd40378e82e99a6542355. As long as we're okay with the `ts-ignore` for now-reachable types

Reply:
Please, see [this suggestion](https://github.com/commercetools/ui-kit/pull/2706/files#r1471390370).